.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - 12
+script:
+  - npm run lint
+  - npm run test
+  - npm run build
+deploy:
+  on:
+    branch: master
+  provider: pages
+  github_token: $GITHUB_DEPLOY_TOKEN
+  local_dir: dist
```

Comment:
Shouldn't it be `docs` rather than `dist` ? Github only allows to publish from a `docs` directory https://help.github.com/en/github/working-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site

Reply:
No, that’s when you’re publishing from the master branch and you want to publish a sub folder. We’re publishing to the church-pages branch, and this option tells the plugin which sub folder from our build we want to publish to the-pages.

Reply:
Indeed, I've missed the `provider: pages`. :disappointed: